daphne/cli.py

```diff
@@ -11,6 +13,7 @@
 
 DEFAULT_HOST = "127.0.0.1"
 DEFAULT_PORT = 8000
+str_or_none = Union[None, str]
```

Comment:
Please don't add typing information without updating the entire file (if you want to do that, a separate PR is probably best)

Reply:
I use these so that my anaconda on sublime will properly autocomplete. Typing is not enforced on python (yet), so I don't see why this would be a problem, since your requirement is already python3.5+, where this is fully supported.

Reply:
It's more that typing information is unrelated to the change as there's not already typing info in the file, so I'd like to keep things consistent and if we add typing do it to the whole file consistently - like if you tried to use `'` quotes not `"`